HISTORY.md

```diff
@@ -2,6 +2,7 @@
 ===================
 
 * Changes from 1.16.0
+* In the next major version, headers will no longer be overwritten when redirecting a directory.
```

Comment:
v2.1.0 was already released. Please add a unreleased section or just add it in the Release PR